File: c/dardo.c
#include <stdio.h>

int main(){
    double distancia1, distancia2, distancia3, maior;

    printf("Digite as tres distancias:\n");
    scanf("%lf %lf %lf", &distancia1, &distancia2, &distancia3);

    if (distancia1 > distancia2 && distancia1 > distancia3) {
        maior = distancia1;
    }
    else if (distancia2 > distancia1 && distancia2 > distancia3) {
        maior = distancia2;
    }
    else {
        maior = distancia3;
    }

	printf("MAIOR DISTANCIA = %.2lf\n", maior);

	return 0;
}

File: python/dardo.py
distancia1: float; distancia2: float; distancia3: float; maior: float

print("Digite as tres distancias:")
distancia1 = float(input())
distancia2 = float(input())
distancia3 = float(input())

if distancia1 > distancia2 and distancia1 > distancia3:
	maior = distancia1
elif distancia2 > distancia1 and distancia2 > distancia3:
	maior = distancia2
else:
	maior = distancia3

print(f"MAIOR DISTANCIA = {maior:.2f}")